docs/src/tutorial.md

```diff
@@ -0,0 +1,137 @@
+# Tutorial
+
+ZX-diagrams are the basic objects in ZX-calculus. In our implementation, each
+ZX-diagram consists of a multigraph and vertices information including type of
+vertices and the phase of vertices. `ZXDiagram` is the data structure for representing
```

Comment:
use `@ref` for the objects you mention in the docs, so one can click them and lead to the actual doc string.

```suggestion
vertices and the phase of vertices. [`ZXDiagram`](@ref) is the data structure for representing
```